support-frontend/assets/pages/paper-subscription-checkout/components/thankYou.jsx

```diff
@@ -107,46 +178,52 @@ function ThankYouContent({
           overheadingClass="--thankyou"
         >
           {isPending ?
-          `Your subscription to the ${cleanProductOption} package is being processed` :
-          `You have now subscribed to the ${cleanProductOption} package`
-  }
+          `Your subscription to the ${cleanProductOption} ${!hasAddedDigitalSubscription ? 'package ' : ''}is being processed` :
```

Comment:
Could extract `${cleanProductOption} ${!hasAddedDigitalSubscription ? 'package ' : ''}` into a const above and use it in both places

Reply:
Thanks, shall do 👍🏻